src/complex-list-field/complex-list-field.component.html

```diff
@@ -42,8 +42,14 @@
         <tr *ngFor="let key of keys[i] | addAlwaysShowFields:schema.items | filterAndSortBySchema:schema.items; trackBy:trackByFunction">
           <td class="label-holder">
             <div>
-              <label>{{key | underscoreToSpace}}</label>
-              <add-new-element-button *ngIf="schema.items.properties[key].type === 'array'" [path]="getValuePath(pIndex, key)" [schema]="schema.items.properties[key]"></add-new-element-button>
+              <title-dropdown [title]="key | underscoreToSpace">
```

Comment:
Actually this was my intention, and I had this on hover background but when I moved the html into `title-dropdown`, somehow it disappeared. However I'll look into it again.